external/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala

```diff
@@ -103,31 +108,49 @@ object SchemaConverters {
       catalystType: DataType,
       nullable: Boolean = false,
       recordName: String = "topLevelRecord",
-      prevNameSpace: String = ""): Schema = {
+      prevNameSpace: String = "",
+      outputTimestampType: AvroOutputTimestampType.Value = AvroOutputTimestampType.TIMESTAMP_MICROS
```

Comment:
do we really need the default value? Seems only one call site excluding the recursive ones.

Reply:
It is also used in `CatalystDataToAvro`